index.html

```diff
@@ -0,0 +1,299 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>My Portfolio Figma 1</title>
+        <link rel="stylesheet" href="./style.css">
+    </head>
+</html>
+<body>
+    <main>
+        <header>
+            <span class="my-logo"><a href='#'>My Logo</a></span>
+            <nav>
+                <a href=""><img src="./assets/images/Union.png" alt="Humburger Button"></a>
+            </nav>
+        </header>
+
+        <section class="hero">
+            <h1 class="greeting">
+                I'm Sio. <br />
+                Glad to see <br />
+                you!
+            </h1>
+            <div class="job-description">
+                <p>I'm a software developer! I can help you build a product , 
+                    feature or website look through some of my work and experience!
+                    If you like what you see and have a project you need coded, 
+                    don't hesitate to contact me.
+                </p>
+            </div>
+            <h2 class="connect-link">LET'S CONNECT</h2>
+            <div class="social-media-icons">
+                <ul class="social media links">
```

Comment:
Hello @May126-sone 👋 ,
You're using the wrong class names in this [line](https://github.com/May126-sone/FIGMA-1-PORTFOLIO/blob/1afffd4c3c53140016a2c852a8c4372e2d02bf2c/style.css#L82C1-L82C1) which makes your styles not applied. Kindly find errors like this in your codes so that every style you apply will work.